app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from dataclasses import dataclass
+from app.errors import (NotVaccinatedError, OutdatedVaccineError,
+                        NotWearingMaskError)
```

Comment:
Use `( )` while importing multiple classes/modules. It is better to avoid using `\` for line continuation. Here is how you can do it: 

python
from app.errors import (
    NotVaccinatedError, 
    OutdatedVaccineError, 
    NotWearingMaskError
)
```